antarest/study/storage/utils.py

```diff
@@ -121,7 +123,21 @@ def extract_output_name(
     path_output: Path, new_suffix_name: Optional[str] = None
 ) -> str:
     ini_reader = IniReader()
-    info_antares_output = ini_reader.read(path_output / "info.antares-output")
+    is_output_archived = path_output.suffix == ".zip"
+    temp_dir = None
+    if is_output_archived:
+        temp_dir = tempfile.TemporaryDirectory()
+        with ZipFile(path_output, "r") as zip_obj:
+            zip_obj.extract("info.antares-output", temp_dir.name)
+            info_antares_output = ini_reader.read(
+                Path(temp_dir.name) / "info.antares-output"
+            )
```

Comment:
maybe add a stopwatch to log the duration (want to check it's not too long).
Same in other places where we read inside the zip